### PR TITLE
#809 - fix browserification

### PR DESCRIPTION
**Pull request by nolanlawson**
My apologies for not adding any tests, but
it seems like you haven't reached the point
where you're browserifying knex and testing
it against WebSQL. If you'd like, though, a
simple command-line test could be:

    browserify . > /dev/null

In any case, I can confirm that after this PR
I am able to browserify knex.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
I'm pretty sure you could replace all of these we either `false` because they are never called, or it would probably be shorter to just right a fricking noop

File: lib/noop.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('lodash/utility/noop')
```
Comment (calvinmetcalf) [inline lib/noop.js]: `module.exports = function noop (){};` is like 1000% times saner to user right here

Comment (nolanlawson) [inline lib/noop.js]: But what if lodash has performance improvements?

Comment (calvinmetcalf) [inline lib/noop.js]: It does not I checked

Comment (nolanlawson) [inline lib/noop.js]: joking, man :)

Comment (calvinmetcalf) [inline lib/noop.js]: I'm not, I wouldn't put it past jdd to have some sort of crazy `new Function` based black magic there

Comment (nolanlawson) [inline lib/noop.js]: hahaha, good point

**Pull request merged**